pkg/machines/machines.less

```diff
@@ -281,3 +281,158 @@ td > .utilization-bar-pf {
         height: 0.75rem;
     }
 }
+
+.error.listing-ct-item {
+    background-color: #fbf0f0 !important;  /* keep red background when expanded */
+}
+
+@toast-stack-offset: 0.75rem;
+@toast-anim-speed: 200ms;
+
+// Notification wrapper styles that apply to both mobile and desktop
+.toast-notification-wrapper {
+    position: relative;
+    display: inline;
+    /* Safari is buggy and needs a prefix for relative stickiness */
+    position: -webkit-sticky;
+    position: sticky;
+    /* Bump notifications so dialogs can pop up on top */
+    z-index: 1049;
```

Comment:
Why this is exact this number isn't really clear to me, but I guess it's due to Bootstrap?
https://stackoverflow.com/a/34181521

Reply:
The number is 1 less than what PatternFly uses for dialogs, inherited from Bootstrap. It's implied in the comment, but I guess, yes, it should be explicit.

`z-index` can be pretty much any whole number up to 2147483647 (32-bit integer) in all modern browsers (older ones are different, but still have absurdly high numbers) — it will work long as position is not static; this is just PF from BS. And, of course, it can also have a negative number, but people usually stop at -1.

But yeah, I'll change the comment.